src/components/layout/playground/submitRequestForm.tsx

```diff
@@ -98,6 +99,10 @@ const SubmitRequestForm: React.FC<SubmitRequestFormParam> = ({ requestId }) => {
     { id: requestId, extended: true },
     {
       enabled: !!requestId,
+      cacheTime: 0,
+      onSuccess: () => {
+        setRequestLoaded(true);
```

Comment:
When setting `requestLoaded` onSuccess, it would in Line 110 already be true, and negate the if condition. This means the useEffect wouldn't execute.